server/task/task-memory/src/test/java/org/apache/james/task/eventsourcing/TaskAggregateTest.java

```diff
@@ -71,7 +72,8 @@ void givenNoStartedTaskShouldEmitNoEventWhenUpdateAdditionalInformationCommand()
             eventId -> Created.apply(ID, eventId, new MemoryReferenceWithCounterTask((counter) -> Task.Result.COMPLETED), HOSTNAME)
         );
         TaskAggregate aggregate = TaskAggregate.fromHistory(ID, history);
-        assertThat(aggregate.update(new MemoryReferenceWithCounterTask.AdditionalInformation(3, timestamp))).isEmpty();
+        assertThat(CollectionConverters.asJava(aggregate.update(new MemoryReferenceWithCounterTask.AdditionalInformation(3, timestamp))))
```

Comment:
in scala?

Reply:
Not for this one, it's used only in scala except for this test